bundles/framework/divmanazer/extension/ExtraFlyout.js

```diff
@@ -97,7 +97,7 @@ Oskari.clazz.define('Oskari.userinterface.extension.ExtraFlyout',
                 } else {
                     me.options.container.append(popup);
                 }
-                popup.find('.icon-close').on('click', function () {
+                popup.find('.oskari-flyouttool-close').on('click', function () {
```

Comment:
Could we have the `.icon-close` element inside the `.oskari-flyouttool-close` element so the bigger element has the click handler and the smaller one is used to clip the icon from the sprite without changes to the icons.css file?